data/test/vtgate/unsupported_cases.txt

```diff
@@ -127,10 +127,6 @@
 "select distinct a, count(*) from user"
 "unsupported: distinct cannot be combined with aggregate functions"
 
-# distinct and group by
```

Comment:
Should you add a non-error test case for this? (If yes, I'm fine if you do this in a separate PR.)